blockchain/blockchain-wallet-v4-frontend
# move fastLink request to bank link component

## Description (optional)
There's a delay when clicking "Link a bank" in order to get the fastlink data from the server and put it into redux. This issue moves the request to the next step so there is no UI pause to the user now.

## Testing Steps (optional)
Click Link a bank in simple buy and you should see the screen transition to the next screen with a loading spinner
